alc/alc.cpp

```diff
@@ -2630,7 +2630,7 @@ START_API_FUNC
             ProbeAllDevicesList();
 
         /* Copy first entry as default. */
-        alcDefaultAllDevicesSpecifier = alcAllDevicesList.c_str();
+        alcDefaultAllDevicesSpecifier = alcAllDevicesList;
```

Comment:
It's not pointless. `alcAllDevicesList` is a null-separated list of names, so the string contains names separated by null characters. A copy assignment will copy the full string, including all the names, while using c_str() invokes the `const char*` assignment, which stops at the first null char and copies just the first name.

Reply:
Removed.